src/reducers/flow.reducer.ts

```diff
@@ -52,6 +61,32 @@ export function reducer(state: State, action: any) {
 			);
 		case FLOWDESIGNER_FLOW_SET_ZOOM:
 			return state.set('transform', action.transform);
+		case FLOWDESIGNER_FLOW_ZOOM_IN:
+			return state.set(
+				'transformToApply',
+				zoomIdentity
+					.translate(state.get('transform').x, state.get('transform').y)
+					.scale(
+						roundByStep(
+							'up',
+							state.get('transform').k + (action.scale || 0.1),
```

Comment:
is 0.1 some accuracy or margin ? put in a dedicated const also

Reply:
can you extract it to the action creator directly? btw I dont get when action of type FLOWDESIGNER_FLOW_ZOOM_IN / OUT might not have this scale property thinking